setup.py

```diff
@@ -4,22 +4,22 @@
 # you may not use this file except in compliance with the Elastic License 2.0.
 #
 import os
+import sys
 
 from setuptools import find_packages, setup
 from setuptools._vendor.packaging.markers import Marker
 
-from connectors.logger import logger
-from connectors.utils import ensure_python_3_10_or_higher
-
 try:
     ARCH = os.uname().machine
```

Comment:
@vidok - thanks for looking at this - I'm concerned a bit still as it looks like this may be the line that is actually causing the original exception to be thrown... defaulting to `x86_64` in most cases I think is safe, however, what will be the consequence of defaulting to this on an `arm64` architecture (as this process is used in both the x76 and aarch64 builds in our processes)?

Reply:
This PR does not change anything but the usage of `connectors.logger`. We just can't use it in setup.py. 

> I'm concerned a bit still as it looks like this may be the line that is actually causing the original exception to be thrown. 

nope, it failed on line 12 because it loads `connectors.logger` from `connectors.utils`. In turn, `connectors.logger`  depends on `ecs-logging` that is not available during the setup.py execution.

Reply:
Yep, it's been there for more than 1 year.